discovery-infra/test_infra/utils.py

```diff
@@ -32,13 +32,14 @@
 conn = libvirt.open("qemu:///system")
 
 
-def run_command(command, shell=False, raise_errors=True):
+def run_command(command, shell=False, raise_errors=True, env=None):
     command = command if shell else shlex.split(command)
     process = subprocess.run(
         command,
         shell=shell,
         stdout=subprocess.PIPE,
         stderr=subprocess.PIPE,
+        env=env,
```

Comment:
IIRC env overwrites the entire env, so it's better to append to it rather than overwrite it completely.

Here is an example of what I mean:
![image](https://user-images.githubusercontent.com/10882062/105873820-8efccd00-6004-11eb-8e91-75caddd73d18.png)